Reorganization of the "Configuring Che" chapter. (#1268)
codenvy/che-docs: src/main/pages/che-7/installation-guide/con_understanding-the-checluster-custom-resource.adoc

[id="understanding-the-checluster-custom-resource_{context}"]
= Understanding the `CheCluster` Custom Resource

A default deployment of {prod-short} consist in the application of a parametrized `CheCluster` link:https://docs.openshift.com/container-platform/latest/operators/crds/crd-managing-resources-from-crds.html[Custom Resource] by the {prod} link:https://docs.openshift.com/container-platform/latest/operators/olm-what-operators-are.html[Operator].

> The `CheCluster` Custom Resource is a YAML document describing the configuration of the {prod-short} cluster.

Reviewer: ```suggestion
The `CheCluster` Custom Resource is a YAML document describing the configuration of the overall {prod-short} installation.
```

> The Operator translates the `CheCluster` Custom Resource into configuration usable by each component: authentication, database, registry, server, storage. 

Reviewer: ```suggestion
The Operator translates the `CheCluster` Custom Resource into configuration usable by each component of the {prod-short} installation: authentication, database, registry, server, storage. 
```

> For instance, to configure the main properties of the {prod-short} server component, the Operator generates the necessary `configMap`, called `che`. Any change in the `configMap` automatically triggers a restart of the {prod-short} Pod.

Reviewer: ```suggestion
For instance, to configure the main properties of the {prod-short} server component, the Operator generates a necessary `ConfigMap`, called `che`. And any change in the `ConfigMap` automatically triggers a restart of the {prod-short} Pod.
```

.Additional resources

* link:https://docs.openshift.com/container-platform/latest/operators/olm-what-operators-are.html[Understanding Operators].

* link:https://docs.openshift.com/container-platform/latest/operators/crds/crd-managing-resources-from-crds.html[Understanding Custom Resources].

> * To learn how to modify the `CheCluster` Custom Resource, see the chosen installation procedure.

Reviewer: I assume in the future you would point to a documentation reference.

Author: Yes, this PR makes visible some missing parts elsewhere, but let's keep the current PR in a reasonable scope.
